src/lib.ts

```diff
@@ -276,52 +277,25 @@ export function printLog(color: string): void {
-    |                          | Recommended to be at least 3501.                                                              |
-    +--------------------------+-----------------------------------------------------------------------------------------------+
-    |      garbo_stashClan     | If set, garbo will attempt to switch to this clan to take and return useful clan stash items, |
-    |                          |  i.e. a Haiku Katana or Repaid Diaper.                                                        |
-    +--------------------------+-----------------------------------------------------------------------------------------------+
-    |       garbo_vipClan      | If set, garbo will attempt to switch to this clan to utilize VIP furniture if you have a key. |
-    +--------------------------+-----------------------------------------------------------------------------------------------+
-    | garbo_skipAscensionCheck | Set to true to skip verifying that your account has broken the prism, otherwise you will be   |
-    |                          |  warned upon starting the script.                                                             |
-    +--------------------------+-----------------------------------------------------------------------------------------------+
-    |  garbo_valueOfFreeFight  | Set to whatever you estimate the value of a free fight/run to be for you. (Default 2000)      |
-    +--------------------------+-----------------------------------------------------------------------------------------------+
-    |     garbo_fightGlitch    | Set to true to fight the glitch season reward. You need certain skills, see relay for info.   |
-    +--------------------------+-----------------------------------------------------------------------------------------------+
-    |       garbo_buyPass      | Set to true to buy a dinsey day pass with FunFunds at the end of the day, if possible.        |
-    +--------------------------+-----------------------------------------------------------------------------------------------+
-    |   garbo_autoUserConfirm  | **WARNING: Experimental** Don't show user confirm dialogs, instead automatically select yes/no|
-    |                          | in a way that will allow garbo to continue executing. Useful for scripting/headless. Risky and|
-    |                          |  potentially destructive.                                                                     |
-    +--------------------------+-----------------------------------------------------------------------------------------------+
-    |           Note:          | You can manually set these properties, but it's recommended that you use the relay interface. |
-    +--------------------------+-----------------------------------------------------------------------------------------------+</pre>`);
+  const helpData = Object.entries(JSON.parse(fileToBuffer("garbo_help.json")));
```

Comment:
Throw a type assertion on here to make your life easier; look at COMBO, for example